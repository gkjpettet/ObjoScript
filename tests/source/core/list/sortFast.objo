var list1 = [6, 4, 2, 8, 9, 0, 0, 1, 3, 3]
list1.sortFast()
System.print(list1) // Expect [0, 0, 1, 2, 3, 3, 4, 6, 8, 9]

var list2 = [6, 4, 2, "a", 9, 0, 0, 1, 3, 3]
list2.sortFast()
System.print(list2) // Expect [0, 0, 1, 2, 3, 3, 4, 6, 9, a]

class Person {}

var list3 = [Person(), 4, 2, "a", 9, 0, 0, 1, 3, 3]
list3.sortFast()
System.print(list3) // Expect [0, 0, 1, 2, 3, 3, 4, 9, a, Person instance]