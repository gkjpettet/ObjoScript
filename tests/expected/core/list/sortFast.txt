[0, 0, 1, 2, 3, 3, 4, 6, 8, 9]
[0, 0, 1, 2, 3, 3, 4, 6, 9, a]
[0, 0, 1, 2, 3, 3, 4, 9, a, Person instance]